commons/src/test/java/org/eclipse/kapua/commons/service/event/store/internal/EventStoreRecordImplTest.java

```diff
@@ -0,0 +1,464 @@
+            assertNotNull("Null not expected.", eventStoreRecordImpl);
+            assertEquals("Expected and actual values should be the same.", scopeId, eventStoreRecordImpl.getScopeId());
+            assertThat("EventStoreRecordImpl object expected.", eventStoreRecordImpl, IsInstanceOf.instanceOf(EventStoreRecordImpl.class));
+        }
+    }
+
+    @Test
+    public void eventStoreRecordImplTest3() throws KapuaException {
+        EventStoreRecordImpl eventStoreRecordImpl = new EventStoreRecordImpl(eventStoreRecord);
+
+        assertThat("EventStoreRecordImpl object expected.", eventStoreRecordImpl, IsInstanceOf.instanceOf(EventStoreRecordImpl.class));
+        assertNotNull("Null not expected.", eventStoreRecordImpl);
+        assertEquals("Expected and actual values should be the same.", "contextId", eventStoreRecordImpl.getContextId());
+        assertEquals("Expected and actual values should be the same.", new Date().toString(), eventStoreRecordImpl.getTimestamp().toString());
+        assertEquals("Expected and actual values should be the same.", new KapuaEid(BigInteger.TEN), eventStoreRecordImpl.getUserId());
+        assertEquals("Expected and actual values should be the same.", "service", eventStoreRecordImpl.getService());
+        assertEquals("Expected and actual values should be the same.", "entityType", eventStoreRecordImpl.getEntityType());
+        assertEquals("Expected and actual values should be the same.", new KapuaEid(BigInteger.TEN), eventStoreRecordImpl.getEntityId());
+        assertEquals("Expected and actual values should be the same.", "operation", eventStoreRecordImpl.getOperation());
+        assertEquals("Expected and actual values should be the same.", "input", eventStoreRecordImpl.getInputs());
+        assertEquals("Expected and actual values should be the same.", "output", eventStoreRecordImpl.getOutputs());
+        assertEquals("Expected and actual values should be the same.", ServiceEvent.EventStatus.TRIGGERED, eventStoreRecordImpl.getStatus());
+        assertEquals("Expected and actual values should be the same.", "note", eventStoreRecordImpl.getNote());
+    }
+
+    @Test
+    public void setAndGetContextIdTest() throws KapuaException {
+        String contextId = "ContextId";
+        KapuaId[] scopeIdList = {null, new KapuaIdStatic(BigInteger.ONE), new KapuaIdStatic(BigInteger.TEN), new KapuaIdStatic(BigInteger.ZERO)};
+
+        for (KapuaId scopeId : scopeIdList) {
+            EventStoreRecordImpl eventStoreRecordImpl1 = new EventStoreRecordImpl(scopeId);
+            assertNull("Null expected.", eventStoreRecordImpl1.getContextId());
+            eventStoreRecordImpl1.setContextId(contextId);
+            assertEquals("Expected and actual values should be the same.", contextId, eventStoreRecordImpl1.getContextId());
+            eventStoreRecordImpl1.setContextId(null);
+            assertNull("Null expected.", eventStoreRecordImpl1.getContextId());
```

Comment:
I think that testing for different values of scopeId is not meaningful.
We want to test only once test and get.

Reply:
Yes good